packages.config

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Abstractions" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.JsonWebTokens" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Logging" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Tokens" version="7.2.0" targetFramework="net48" />
+  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net48" />
```

Comment:
We already have System.text.JSON -- we should only use one package and migrate off of newtonsoft ( https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-9-0)

Reply:
done